src/net_processing.cpp

```diff
@@ -779,6 +779,23 @@ void PeerLogicValidation::InitializeNode(CNode *pnode) {
         PushNodeVersion(pnode, connman, GetTime());
 }
 
+// We don't force an attempt to relay unbroadcast txns by directly adding them
+// to setInventoryTxToSend, but instead pass to PushInventory which checks
+// filterInventoryKnown.
+//
+// Passing txns through the bloom filter has a stronger privacy model, but weaker
+// delivery guarantees. Under certain network & node conditions, it may still
+// take an extended period of time for the transaction to be initially broadcast
+// to the network.
```

Comment:
in commit 2f3f449:

I don't understand this comment, why would it have weaker delivery guarantees? Is it the bloom filter false positive rate?

Reply:
no, simpler.. just mean some transactions will get filtered & not relayed because of `filterInventoryKnown`. this was a result of the confusion & conversation in https://github.com/bitcoin/bitcoin/pull/18038#discussion_r385865086. 

but maybe without that context this comment is misleading? appreciate any feedback on how to clarify. right now I'm thinking of just deleting the second paragraph.

Reply:
Why would `filterInventoryKnown` return a match when the transaction was initially never sent out to that peer?

Reply:
eg. if you disconnect wifi & submit a txn to your mempool. code will add to the bloom filters but the transaction never actually went anywhere

Reply:
updated this so comment above `ReattemptInitialBroadcast` just notes that unbroadcast txs will be passed through `filterInventoryKnown`. Leaving the tradeoff analysis for reviewers / future people writing code.